components/QuizPage.styled.js

```diff
@@ -1,53 +1,54 @@
 import Link from "next/link";
 import styled from "styled-components";
+import Image from "next/image";
 
 export const StyledLink = styled(Link)`
   text-decoration: none;
   color: var(--color-fox);
-  font-size: 1.5rem;
-  margin-bottom: 1rem;
+  font-size: 1.3rem;
+  margin-bottom: 0.5rem;
 `;
 
 export const StyledContainer = styled.div`
   display: flex;
   flex-direction: column;
+  flex: 1;
+  justify-content: space-around;
+`;
+
+export const StyledContainerWrapper = styled.div`
+  flex: 1;
+  position: relative;
+
+  /* CardContainer has a margin-bottom of 3rem for rendering in lists, here we reduce the margin to the quiz navigation */
```

Comment:
Yes, I first tried as you said, but it has broken the complete layout.
Since we are all not designers in group, we unfortunately missed to predefine a few basics in the beginning and by now we can hardly touch layout issues without refactoring the big picture to keep it working/looking good. For a bigger refactoring as it is needed in this case, there's not enough time left anymore. So we keep this working solution, as we won't use this components another time and for now there're no problems with it.